### PR TITLE
CAT-1154 Added Dialog for choosing new Sound File from Recorder or Ga…

### PR DESCRIPTION
**Pull request by loibi93**
…lery

CAT-1154 Hopefully fixing Lint error

Fixed Lint error and small Bug

CAT-1154 Fixed three tests

CAT-1154 Fixing rebase errors

CAT-1154 Fixed two tests, one to go

CAT-1154 Fixed Bug when returning from SoundFragment to ScriptFragment with result.

CAT-1154 Fixed typo.

### CODE REVIEW

**Comment by loibi93** [comment]
https://jenkins.catrob.at/job/Catroid-Multi-Job-Custom-Branch-RELOADED/2139/

https://jenkins.catrob.at/view/All-Categories/view/Catroid/job/Catroid-Device-Single-ClassAndPackage-Test/332/

https://jenkins.catrob.at/view/All-Categories/view/Catroid/job/Catroid-Device-Single-ClassAndPackage-Test/333/

https://jenkins.catrob.at/view/All-Categories/view/Catroid/job/Catroid-CustomEmulator-Single-ClassAndPackage-Test/406/

Tests run fine.

**Comment by thmq** [comment]
looks pretty good. although the commit messages should be reworded according to the guidelines here.

**Comment by loibi93** [comment]
That was on purpose ;) I always wait until someone approves the PR and then i do the final reword, because otherwise i always have to rewrite it, and that way I can have a final look at all my commits and write something useful but short !

**Comment by loibi93** [comment]
Reworded Commit Message and rebased. Do I have to test again ?

**Comment by loibi93** [comment]
Testrun:
https://jenkins.catrob.at/view/All-Categories/view/Catroid-multi-job/job/Catroid-Multi-Job-Custom-Branch-RELOADED/2185/#showFailuresLink

Singletest of failed tests (except UserBrickScriptActivity since we know that this is failing quite often)
https://jenkins.catrob.at/view/All-Categories/view/Catroid/job/Catroid-Device-Single-ClassAndPackage-Test/348/
https://jenkins.catrob.at/view/All-Categories/view/Catroid/job/Catroid-CustomEmulator-Single-ClassAndPackage-Test/418/
https://jenkins.catrob.at/view/All-Categories/view/Catroid/job/Catroid-Device-Single-ClassAndPackage-Test/350/

**Comment by loibi93** [comment]
ok to test

**Comment by loibi93** [comment]
ok to test

**Comment by thmq** [comment]
I really don't like to be a pain in your .. workflow, but could you reword the commit message to something more concise? something like: 

```
CAT-1154 Added Dialog for new Sound

Fixed return to ScriptFragment from SoundFragment
```

**Comment by loibi93** [comment]
ok to test

**Comment by loibi93** [comment]
ok to test

**Comment by loibi93** [comment]
setLookBrickTest keeps failing on Jenkins, but runs fine on my local Device, I will have to investigated that issue.

File: catroid/src/org/catrobat/catroid/ProjectManager.java [hunk 1]
```diff
@@ -71,20 +71,40 @@
...
 
-	public void setCommingFromScriptFragment(boolean value) {
-		this.commingFromScriptFragment = value;
+	public void setCommingFromScriptFragmentToSoundFragment(boolean value) {
+		this.commingFromScriptFragmentToSoundFragment = value;
 	}
 
-	public boolean getCommingFromScriptFragment() {
-		return this.commingFromScriptFragment;
+	public boolean getCommingFromScriptFragmentToSoundFragment() {
```
Comment (aried3r) [inline catroid/src/org/catrobat/catroid/ProjectManager.java]: `Coming` with only one "m". Same for the methods below.

**Pull request closed without merging**